.travis.yml

```diff
@@ -53,10 +64,24 @@ addons:
       - groovy
 
 before_script:
+    - |
+      if [[ $TRAVIS_CPU_ARCH == 'ppc64le' ]]; then
+          export JAVA_HOME=/usr/lib/jvm/adoptopenjdk-8-hotspot-ppc64el
+          export PATH=$JAVA_HOME/bin:$PATH
+      fi
     - groovy --version
 
 script:
-  - xmlstarlet ed --inplace -d "//mirrors" /home/travis/.m2/settings.xml
+  - |
+    set -e
+    MVN_SETTINGS=${TRAVIS_HOME}/.m2/settings.xml
+      if [[ -f ${MVN_SETTINGS} ]]; then
```

Comment:
Technically this shouldn't be indented since the previous line isn't an if or such.

Reply:
fixed